app/src/main/java/org/pesmypetcare/mypetcare/controllers/community/TrUnbanPost.java

```diff
@@ -0,0 +1,57 @@
+package org.pesmypetcare.mypetcare.controllers.community;
+
+import org.pesmypetcare.mypetcare.features.community.forums.NotForumOwnerException;
+import org.pesmypetcare.mypetcare.features.community.posts.Post;
+import org.pesmypetcare.mypetcare.features.users.User;
+import org.pesmypetcare.mypetcare.services.community.CommunityService;
+
+/**
+ * @author Xavier Campos
+ */
+public class TrUnbanPost {
+    private CommunityService communityService;
+    private User user;
+    private Post post;
+    private boolean result;
+
+    public TrUnbanPost(CommunityService communityService) {
+        this.communityService = communityService;
+    }
+
+    /**
+     * Setter of the user that wants to unban the post.
+     * @param user
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [At-clause should have a non-empty description.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)